capability.go

```diff
@@ -853,8 +819,8 @@ type ClientHook interface {
 
 	// Shutdown releases any resources associated with this capability.
 	// The behavior of calling any methods on the receiver after calling
-	// Shutdown is undefined.  It is expected for the ClientHook to reject
-	// any outstanding call futures.
+	// Shutdown is undefined. Shutdown must not interrupt any already
```

Comment:
Small nit/question:  is this a hand-wavy "must" or an RFC 2119 "MUST"?  I have code wherein long-running goroutines detect the canceled context upon shutdown, and abort.  That seems like it doesn't technically fall under "shutdown interrupted a call", but rather under "a call detects a shutdown and bows out", but I want to make sure this is okay.

If it is okay, then I think the docstring could use a bit of clarification.

Reply:
This is part of #502. See https://github.com/capnproto/go-capnp/pull/502#discussion_r1164318648. We could all-caps it I suppose.

Reply:
Right, it did seem familiar!  No need to all-caps it, I was just referring to RFC 2119 to distinguish from "you probably won't need to bother with that".

It's being handled to my satisfaction in #502, so I think this is ready to merge. 👍